Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -70,8 +70,9 @@ class DropDownMenu_ItemInfo
       public IsDisabled: boolean,
       public readonly Href: Nullable<string>,
       public readonly Target: Nullable<string>,
+      public readonly FallbackNavigationUrl: string,
       public readonly DiagnosticMetadata: Nullable<Dictionary<string | boolean>>,
-      public readonly DiagnosticMetadataForCommand: Nullable<Dictionary<string | boolean>>)
+      public readonly DiagnosticMetadataForCommand: Nullable<Dictionary<string | boolean>>)  
```

Comment:
trailing whitespace.

What I try to do is review the entire changeset when I do a commit to find lines that should not contain changes or have a formatting bug, left over TODO commetns etc